### PR TITLE
centos-10 fixes

### PR DESCRIPTION
**Pull request by martinpitt**
See https://github.com/cockpit-project/cockpit/pull/20321

 * [x] image-refresh centos-10

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh centos-10 done: https://github.com/cockpit-project/bots/commits/image-refresh-centos-10-20240417-074756

**Review by jelly — APPROVED** [review]

**Comment by martinpitt** [comment]
Argh -- the firewalld (2.1.1-1.el10 -> 2.1.2-1.el10) update [is kaputt](https://cockpit-logs.us-east-1.linodeobjects.com/pull-6235-9d96177a-20240417-075023-centos-10-cockpit-project-starter-kit/log.html):
```
# firewall-cmd --add-service=cockpit --permanent
ERROR:dbus.proxies:Introspect error on :1.6:/org/fedoraproject/FirewallD1: dbus.exceptions.DBusException: org.freedesktop.PolicyKit1.Error.Failed: Action org.fedoraproject.FirewallD1.info is not registered
Error: Action org.fedoraproject.FirewallD1.info is not registered
```
This is rather weird -- `rpm -ql firewalld` claims that /usr/share/polkit-1/actions/org.fedoraproject.FirewallD1.policy is part of the RPM, but that file doesn't exist. In the previous image it does. 

Downloading and inspecting the rpm shows that it's supposed to be a symlink:

> /usr/share/polkit-1/actions/org.fedoraproject.FirewallD1.policy -> org.fedoraproject.FirewallD1.server.policy.choice

but it doesn't exist after installation. The workaround is simple enough:
```
ln -s org.fedoraproject.FirewallD1.server.policy.choice /usr/share/polkit-1/actions/org.fedoraproject.FirewallD1.policy
```
I'll report it to jira and apply it.

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-centos-10-92478784-20240417-081340/log.html

**Comment by cockpituous** [comment]
image-refresh centos-10 done: https://github.com/cockpit-project/bots/commits/image-refresh-centos-10-20240417-083453

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-centos-10-92478784-20240417-081946/log.html

File: images/scripts/rhel.setup [hunk 1]
```diff
@@ -184,6 +184,7 @@ sssd-dbus \
 "
 
 TEST_PACKAGES="\
+acl \
```
Comment (allisonkarlitskaya) [inline images/scripts/rhel.setup:187]: Do we know for sure that's on all RHEL?

**Review by allisonkarlitskaya — APPROVED** [review]

File: images/scripts/rhel.setup (hunk 1 above)
Comment (martinpitt) [reply images/scripts/rhel.setup:187]: Yes, all the way back to RHEL 3. It just used to be part of the base system (systemd dependency) until 9, so we never installed it explicitly.

**Pull request merged**